templates/katello.yaml.erb

```
@@ -23,19 +23,10 @@
     :ssl_key_file: <%= @candlepin_events_ssl_key %>
     :ssl_ca_file: <%= @candlepin_ca_cert %>
 
+  :katello_applicability: true
+
 <% if @pulp2_support %>
```

Small nit: you get cleaner output if you swap it around:
```suggestion
<% if @pulp2_support %>

```

The net result is that for users where `@pulp2_support` is false, is that the file doesn't change for them. That means it may not have to restart Foreman(/Katello).

---

I do not think we have any users for which that is true right now, and this file will undergo further changes in the end to support Katello 4.0, so I will hold off further changes.